<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Break</class>
 <widget class="QWidget" name="Break">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>419</width>
    <height>68</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(110, 110, 110, 255), stop:0.101942 rgba(91, 91, 91, 255), stop:0.820388 rgba(60, 60, 60, 255), stop:0.990291 rgba(37, 37, 37, 255));
	color: rgb(225, 225, 225);
}
QTextBrowser, QLineEdit {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(37, 37, 37, 147), stop:0.0873786 rgba(60, 60, 60, 150), stop:0.932039 rgba(91, 91, 91, 148), stop:1 rgba(110, 110, 110, 148));
	border: 1px  solid rgb(147, 45, 203);
	border-radius: 3px;
}

QLCDNumber, QSpinBox, QDateTimeEdit {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(37, 37, 37, 147), stop:0.0873786 rgba(60, 60, 60, 150), stop:0.932039 rgba(91, 91, 91, 148), stop:1 rgba(110, 110, 110, 148));
	border: 1px  solid rgb(44, 44, 44);
	border-radius: 3px;
}

QLabel {
	background-color:rgba(255, 255, 255, 0);
}

QPushButton {
	background-color:rgba(255, 255, 255, 0);
}

QProgressBar {
	background-color: rgb(44, 44, 44);
	border: 1px  solid rgb(44, 44, 44);
	margin: 8px, 8px;
	color:rgba(255, 255, 255, 0);
}
QProgressBar::chunk {
	background-color: rgb(147, 45, 203);
}</string>
  </property>
  <widget class="QPushButton" name="BreakButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>115</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Done</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="BreakTime">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::UpDownArrows</enum>
   </property>
   <property name="correctionMode">
    <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
   </property>
   <property name="suffix">
    <string> min</string>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>20</number>
   </property>
  </widget>
  <widget class="QProgressBar" name="BreakProg">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>151</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
